src/collection-view.js

```diff
@@ -482,7 +487,13 @@ Marionette.CollectionView = Marionette.View.extend({
 
   // render the child view
   renderChildView: function(view, index) {
+    if (!(view instanceof Marionette.View)) {
+      Marionette.triggerMethodOn(view, 'before:render', view);
+    }
     view.render();
+    if (!(view instanceof Marionette.View)) {
```

Comment:
The only other hook we test for in core currently is the absence of a `destroy` method.  The only thing I don't like about the destroy method check is that it's not obvious what we're asking.  Is `_isMarionetteView` more performant than an instanceof check?  I _could_ add something like that into this commit.